docs-kits/kits/Connector Kit/Adoption View/working-with-policies.md

```diff
@@ -0,0 +1,120 @@
+`odrl:permission`, `odrl:prohibition` and `odrl:obligation` properties. They hold `odrl:Rules` that have to be
+satisfied by the Consumer. There's two ways that can happen.
+
+### Consumer-side: `odrl:Offer` in a `ContractRequestMessage`
+
+Firstly, when transmitting a [`ContractRequestMessage`](https://docs.internationaldataspaces.org/ids-knowledgebase/v/dataspace-protocol/contract-negotiation/contract.negotiation.protocol#id-2.1-contract-request-message)
+(for example triggered via the EDC's [contract-negotiations API](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/docs/usage/management-api-walkthrough/05_contractnegotiations.md)), the
+Consumer must signify consent to the Offer by including it in the Request and adding the `odrl:target` property on the
+level of an `odrl:Offer`. It is a very, very bad idea to simply copy the next-best Offer from the Catalog and replay it
+to the Provider. Instead, Consumers must perform internal checks which (if any) Offer proposes terms that they want to
+agree to. It is possible (though not usual) to encode pricing information in an `odrl:Offer`. Sending a
+`ContractRequestMessage` echoing the pricing information would represent a legally binding agreement in accordance to
+the conditions proposed by the Provider's Offer. Thus, selecting an offer extended by a Provider company should thus be
+handled with appropriate care.
+
+The Provider will usually accept the ContractRequest only if the extended Offer is semantically equivalent to his own
+as distributed via the Catalog.
+
+### Provider-side: Checking a Consumer's Verifiable Presentation
+
+In addition to checking the Offer's structure (as above), Providers' Connectors have the chance to query the Consumer's
+Verifiable Presentation (VP). A VP is a set of Verifiable Credentials (VCs) that a Consumer may have been issued by a
+trusted third party. In this PR, a Provider may find additional information if a Consumer is eligible to pass a certain
+`odrl:Constraint`.
+
+If, for example, the Consumer tries to negotiate for an Offer that is extended only to interested
+parties from civil society (like an NGO), simply pretending to be an NGO shouldn't be enough. It has to be verified and
+can be verified with help of the VP. For that, a "NonGovernmentalOrganizationCredential" could be issued during the
+onboarding process according to the role a Participant assumes in the Dataspace. For more on the interrelation between
+Credentials and Constraints, refer to standards CX-0018 and CX-0149.
+
+A offer containing a `odrl:Use` permission with a `odrl:Constraint` could (in a hypothetical scenario) look like this:
+
+```json
+{
+  "@context": {
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "@id": "Y29udHJhY3QtYmlsYXRlcmFsLXBhcGVyLWV4YW1wbGUtNA==:anNvbi0xLXBhcGVy:Y2JhMGFmM2QtZTIwYi00ZjExLThlZmQtM2FhMzcyZmI1Nzhh",
+  "@type": "odrl:Set",
+  "odrl:permission": {
+    "odrl:action": {
+      "odrl:type": "http://www.w3.org/ns/odrl/2/use"
```

Comment:
'odrl:type' -> '@id'
as mentioned, this is a bug. Not sure if we should add bug behavior to the documenation.

Reply:
will fix